zombie_dice/ai.go

```diff
@@ -0,0 +1,24 @@
+package zombie_dice
+
+import (
+	"github.com/Akavall/GoGamesProject/dice"
+)
+
+func GreedyAI(shots int) int {
```

Comment:
I think I see how interface will help.

select_ai() will return an ai_object instead of a string. In PlayWithAI() will pass ai_object to players_turn() (nil for human player), and inisde of players_turn() we'll call ai_object.shouldKeepGoing() (if ai_obect is not nil). This eliminates the switch statement inside players_turn().